### PR TITLE
Added fiew Interfaces, Entities and created first post and get reques…

### PR DESCRIPTION
**Pull request by sereczeq**
…ts for /tests

### CODE REVIEW

File: backend/src/api/test/entities/test.entity.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export class TestEntity implements TestInterface
+{
+    @PrimaryGeneratedColumn()
+    id: number;
+
+    @ManyToOne(() => SettingsEntity, settings => settings.tests, {cascade :true})
+    settings: SettingsInterface;
+
+    @ManyToOne(() => OwnerEntity, owner => owner.tests, {cascade :true})
+    owner : OwnerInterface;
```
Comment (Kartm) [inline backend/src/api/test/entities/test.entity.ts]: Do we need to store a `owner`? I thought we'll only send and save `owner_link`

Comment (sereczeq) [reply backend/src/api/test/entities/test.entity.ts]: Nice catch. But that's taken straight from conceptual diagram. Maybe that needs a redo

Comment (Kartm) [reply backend/src/api/test/entities/test.entity.ts]: From this one? 
image

Comment (sereczeq) [reply backend/src/api/test/entities/test.entity.ts]: No, I mean this
image

**Review by Kartm — APPROVED** [review]
Great work!

**Pull request merged**